db/projects/pundix/project.yaml

```diff
@@ -0,0 +1,74 @@
+name: Pundi X
+web:
+  - 'https://pundix.com/'
+start_date: '2016-01-01'
+description: 'Pundi X aims to bring in the next billion crypto users as it allows users to buy, sell, use cryptocurrency anywhere and anytime.'
```

Comment:
Maybe I agree with Tree in this opinion. _aims to bring in the next billion crypto users_ This is nothing like description, but more like their advertise. But anyway, I know how hard is to write some normal description. So maybe better not to fill this field?

Reply:
I changed description to "Zilliqa is project focused on scalability and sharding research"